jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/infocommands/InfoThreadCommand.java

```diff
@@ -284,15 +285,13 @@ private boolean printThreadInfo(String id, Map threads)
 	public void printRegisters(ImageThread it)
 	{
 		out.print("   registers:");
-		out.print("\n");
-		
 		int count = 0;
 		Iterator itImageRegister = it.getRegisters();
 		while (itImageRegister.hasNext()) {
 			if (count % 4 == 0) {
-				if (0 != count) out.print("\n");
-				out.print("    ");
+				out.print("\n  ");
 			}
+			out.print("  ");
```

Comment:
This seems a change in behavior, printing 2 spaces between registers when before it was 3.

Reply:
Yes. Do you think the extra space is helpful?

Reply:
any change is at risk of breaking something.

Reply:
Spacing reverted to 3.